### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kjw000**

### CODE REVIEW

**Review by tdaly61 — APPROVED** [review]
looks ok to me

File: profile/README.md [hunk 1]
```diff
@@ -4,3 +4,7 @@
     <img src="https://raw.githubusercontent.com/mojaloop/.github/main/profile/mojaloop_logo.png"/>
   </a>
 </p>
+# Welcome to the Mojaloop Open Source Site on Github
+
+* Review the documentation: http://mojaloop.io/documentation
+* Join the Conversation on Slack: https://mojaloop.io/slack
```
Comment (elnyry-sam-k) [inline profile/README.md:10]: Can we maybe put this back when the herokuapp is working?

Comment (elnyry-sam-k) [reply profile/README.md:10]: until then we can use this instead: https://join.slack.com/t/mojaloop/shared_invite/zt-1qy6f3fs0-xYfqfIHJ6zFfNXb0XRpiHw

**Review by elnyry-sam-k — CHANGES_REQUESTED** [review]
Thanks for raising this, Kim! 

provided details in comments

**Pull request merged**